perfmetrics/scripts/testing_on_gke/examples/utils/utils_test.py

```diff
@@ -34,51 +34,29 @@ def setUpClass(self):
     self.start = '2024-09-25 06:32:22 UTC'
     self.end = '2024-09-25 07:06:22 UTC'
 
-  def test_get_memory_methods(self):
-    low1, high1 = get_memory_from_monitoring_api(
+  def test_get_memory(self):
+    low, high = get_memory_from_monitoring_api(
         project_id=self.project_id,
         cluster_name=self.cluster_name,
         pod_name=self.pod_name,
         namespace_name=self.namespace_name,
         start_epoch=self.start_epoch,
         end_epoch=self.end_epoch,
     )
-    self.assertLessEqual(low1, high1)
-    self.assertGreater(high1, 0)
+    self.assertLessEqual(low, high)
```

Comment:
Add empty line between Act and Assert blocks. Ditto for the other test.

Reply:
done